lib/game.rb

```diff
@@ -23,6 +24,6 @@ def take_turn
   end
 
   def end_game
-    @current_player.end_game(@checker.winner)
+    @current_player.end_game(@checker.winner, @player1.token)
```

Comment:
This works because the player 1 is always the human player, yeah?

It feels a little off that we would assign `player1` to two different instance variables.

Reply:
Removed this assignment and presented the winner in another fashion.